dotnet/src/Connectors/Connectors.Memory.Qdrant/Diagnostics/ValidateExtensions.cs

```diff
@@ -8,12 +8,12 @@ namespace Microsoft.SemanticKernel.Connectors.Memory.Qdrant.Diagnostics;
 
 internal static class ValidateExtensions
 {
-    public static void Validate(IValidatable target)
+    public static void Validate(IValidatable? target)
     {
         target?.Validate();
     }
 
-    public static void Validate(params IValidatable[] targets)
+    public static void Validate(IEnumerable<IValidatable>? targets)
```

Comment:
This actually changes the calling semantics, no? What resharper rule triggered this? We may want to disable it.

Reply:
reverted in [that commit](https://github.com/microsoft/semantic-kernel/commit/8b0eb80f816715c5aecb81d9ec56457016c5c82c).